src/Microsoft.HttpRepl.IntegrationTests/Mocks/MockedShellState.cs

```diff
@@ -10,31 +10,33 @@ namespace Microsoft.HttpRepl.IntegrationTests.Mocks
 {
     internal class MockedShellState : IShellState
```

Comment:
@bhsubra I refactored this so that less of it was mocked. I don't know if this will break any of your current PRs, but it works with everything that's currently in master.

Reply:
I verified it locally. This won't break any of the changes that I have in pr or locally. Thanks!